joomla/joomla-cms
# [5.2][Events] Use event classes for Finder plugins

### Summary of Changes

Update `finder` plugins to use `SubscriberInterface` and an event classes.


### Testing Instructions

Run finder indexer, edit the content. 
Check the indexer works, and changed content also indexed.


### Actual result BEFORE applying this Pull Request

Works


### Expected result AFTER applying this Pull Request

Works


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/177
